CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## v1.5.2
+
+* Enhancements
+  * Added `NERVES_DATA_DIR` environment variable for specifying the location
```

Comment:
I think this one is just:

  * Support `XDG_DATA_HOME`. If `XDG_DATA_HOME` is set, Nerves will now store its data under that directory.

This feels like the least important enhancement, so my vote would be to list it last. The `erlinit` one feels like the most important one or at least the one that's caused the most annoyance.